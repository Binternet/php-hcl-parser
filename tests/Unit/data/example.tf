
# Specify the provider and access details
provider "aws" {
  access_key = "${var.access_key}"
  secret_key = "${var.secret_key}"
  region     = "${var.region}"
}

# Create an instance
resource "aws_instance" "example" {
  ami           = "ami-2757f631"
  instance_type = "t2.micro"
}
