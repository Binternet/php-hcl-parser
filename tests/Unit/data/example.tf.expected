O:8:"stdClass":2:{s:8:"provider";a:1:{i:0;O:8:"stdClass":1:{s:3:"aws";a:1:{i:0;O:8:"stdClass":3:{s:10:"access_key";s:17:"${var.access_key}";s:6:"region";s:13:"${var.region}";s:10:"secret_key";s:17:"${var.secret_key}";}}}}s:8:"resource";a:1:{i:0;O:8:"stdClass":1:{s:12:"aws_instance";a:1:{i:0;O:8:"stdClass":1:{s:7:"example";a:1:{i:0;O:8:"stdClass":2:{s:3:"ami";s:12:"ami-2757f631";s:13:"instance_type";s:8:"t2.micro";}}}}}}}